cmd/helm/lint.go

```diff
@@ -51,19 +51,33 @@ func init() {
 	RootCommand.AddCommand(lintCommand)
 }
 
-var errLintNoChart = errors.New("no chart found for linting (missing Chart.yaml)")
-var errLintFailed = errors.New("lint failed")
+var errLintNoChart = errors.New("No chart found for linting (missing Chart.yaml)")
```

Comment:
Oh, usually it says something about not starting an error with a capitalized word.